packages/text-sets/src/raw/step.json

```diff
@@ -1880,7 +1880,7 @@
           "lockAspectRatio": false,
           "backgroundTextMode": "NONE",
           "font": {
-            "family": "Open Sans Condensed",
+            "family": "Roboto Condensed",
```

Comment:
In the issue I see:

> Roboto Condensed, Light  

If that meant to be  Roboto Condensed Light  

Does the font weight need to be updated to 300 ?

Reply:
> Does the font weight need to be updated to 300 ?

Yup 👍